extensions/astro/languages/astro/highlights.scm

```diff
@@ -1,7 +1,7 @@
 (tag_name) @tag
 (erroneous_end_tag_name) @keyword
-(doctype) @constant
-(attribute_name) @property
+(doctype) @tag
```

Comment:
The use of `@constant` for the doctype is intentional, as we want it to be visually distinct from the other tags.

Reply:
If we do want to change it we could do something like `@tag.doctype` to allow themes to individually target it, if desired.